recipes/vmd-python/meta.yaml

```diff
@@ -0,0 +1,66 @@
+{% set name = "vmd-python" %}
+{% set version = "2.0.10" %}
+{% set sha256 = "ac25c7291208868be51ef1cbb9235e70b0b96f30fcd2ec5eb3af3e31ebd66ca9" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-v{{ version }}.tar.gz
+  url: https://github.com/Eigenstate/{{ name }}/archive/v{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: true  # [win]
+  script: python -m pip install --no-deps --ignore-installed .
+
+requirements:
+  build:
+    - python
+    - pip
+    - perl
+    - make
+    - toolchain
+    - nomkl
+    - libnetcdf 4.4.*
+    - expat
+    - numpy >=1.13
+    - tk 8.5.*
+
+  run:
```

Comment:
Do you want a build and run dep of `sqlite 3.20.*` also? Looks like [it's used](https://github.com/Eigenstate/vmd-python/blob/v2.0.10/setup.py#L262-L265), and 3.20 [is the current conda-forge preferred version](https://github.com/conda-forge/conda-forge-pinning-feedstock/blob/2999590b6fbdd908dac5a6f85f79b25c43b33bf4/recipe/conda_build_config.yaml#L199-L200).